docs/howtos/optax_update_guide.rst

```diff
@@ -0,0 +1,263 @@
+Optax Update Guide
```

Comment:
As it reads now it sounds like it's about updating Optax.

I propose renaming to "Upgrading my codebase to Optax"